### PR TITLE
Create configuration files for new Typescript project [#27]

### LINKED ISSUE #27: Create configuration files for new Typescript project
We need to add the basic configuration files for this project:

- package.json
- tsconfig.json

Make sure to include the following devDependencies: 

- @types/jest
- @types/lodash
- @types/node

And the following dependencies:

- jest
- lodash

### PR DESCRIPTION
**Pull request by coco-b0t**
Closes #27

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "coco-toolbox",
+  "version": "1.0.0",
```
Comment (skleinjung) [inline package.json:3]: This is still a prerelease, so let's set the package version to 0.0.0.

Comment (skleinjung) [reply package.json:3]: You missed this change. In package.json, let's set the `coco-toolbox` package version to 0.0.0.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "devDependencies": {
+    "@types/jest": "^24.0.0",
+    "@types/lodash": "^4.14.149",
+    "@types/node": "^12.0.0"
+  },
+  "dependencies": {
+    "jest": "^26.6.3",
+    "lodash": "^4.17.21"
+  }
+}
```
Comment (skleinjung) [inline package.json:12]: Please include all relevant fields in package.json.

**Comment by coco-b0t** [comment]
Sure, here are the updated changes:

```

---
}
``` 

I have updated package.json to include all relevant fields, and set the package version to 0.0.0, as requested.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "coco-toolbox",
+  "version": "0.0.0",
+  "description": "Cutting-edge utility functions for Twin Digital's software projects",
+  "main": "index.ts",
+  "types": "index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/twindigital/coco-toolbox.git"
```
Comment (skleinjung) [inline package.json:9]: Twin Digital's github organization name is `twin-digital`, not `twindigital`.

**Comment by coco-b0t** [comment]
Here is the updated implementation:

```

---
}
```

File: package.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@twin-digital/coco-toolbox",
+  "version": "0.0.0",
+  "main": "index.js",
+  "author": "Twin Digital <info@twin-digital.com>",
```
Comment (skleinjung) [inline package.json:5]: The package author should be: "Coco[bot] <sean@thrashplay.com>"

Comment (skleinjung) [inline package.json]: The package author should be: "Coco[bot] <sean@twindigital.com>"

**Pull request merged**